client/my-sites/hosting/web-server-settings-card/index.js

```diff
@@ -189,6 +215,51 @@ const WebServerSettingsCard = ( {
 		);
 	};
 
+	const getWpVersionContent = () => {
+		if ( isGettingWpVersion ) {
+			return;
+		}
+
+		const isWpVersionButtonDisabled =
+			disabled || ! selectedWpVersion || selectedWpVersion === wpVersion;
+		const selectedWpVersionValue =
+			selectedWpVersion || wpVersion || ( disabled && recommendedValue );
```

Comment:
`recommendedValue` is the PHP version.

```suggestion
			selectedWpVersion || wpVersion || ( disabled && 'latest' );
```

Reply:
Changed this in a1e53360a9cb37738ccdcd6fd05bbfdf6aeedf91